presenter/userFormPresenter.vue

```diff
@@ -0,0 +1,104 @@
+<script
+    setup
+    lang="tsx">
+    import { signInWithEmailAndPassword, createUserWithEmailAndPassword, signOut } from 'firebase/auth'
+    
+    import LoginView from "~/views/loginView.vue";
+    import SignupView from "~/views/signUpView.vue";
+
+    const items = [{
+      key: 'login',
+      label: 'Log in'
+    }, {
+      key: 'signup',
+      label: 'Sign up'
+    }]
```

Comment:
Do you think there is a shorter way to do this ? Maybe "hard-coding" these names in the code below ? But otherwise I trust you, don't change that ! I am not very aware of Vue ways to do `login` / `logout` forms.